meetings/2024-04-10/minutes.md

```diff
@@ -4,36 +4,102 @@
+ - [x] [Ry Jones (Linux Foundation)](https://github.com/ryjones)
+ - [x] [Hart Montgomery, Linux Foundation](https://github.com/hartm)
+
+
 ## 1. Chair's introduction
+ - The chair thanks members for being patient as we get up and running
+ - TSC aims to be consensus driven & friendly
+ - TSC will discuss & make strategic decisions on technical matters. Developer call handlers regular operational matters
 
 ## 2. Approve agenda
 
+ - Agreed
+
 ## 3. Appoint minute-taker
 
+ - Nigel will take this week's notes. Agreed that we would ask at the beginning of each meeting & do in rotation.
+
 ## 4. Approval of minutes of last meeting
 
+- Last meeting's notes are open as [PR #14](https://github.com/open-quantum-safe/tsc/pull/14). Comments/reviewers invited. TSC members asked to approve. 
+- will use direct links to recordings in future minutes - portal not available to all (has been requested)
+
 ## 5. Voting procedure
 
+- [Issue #12](https://github.com/open-quantum-safe/tsc/issues/12) open for discussion
+- online/async voting to allow broadest engagement
+- open votes for technical matters
+- differing opinions on whether personal matters (including voting such as ranking) should be public or secret, or referred up to the board.
+- noted that most decisions consensus driven in any case
+- mechanism - git (simple), [gitvote](https://github.com/cncf/gitvote), [Helios](https://vote.heliosvoting.org/) are possible options
+- discussion to continue in [issue 12](https://github.com/open-quantum-safe/tsc/issues/12)
+
+
 ## 6. Addition of Vlad Gheorghiu to TSC
 
+- Vlad is maintainer of language wrappers 
+- Intent had been he was in the TSC from launch
+- Christan proposed, Basil seconded motion to add Vlad. Agreed by verbal vote of TSC members with no objections
+
 ## 7. Meeting cadence
 
+- Monthly still seems appropriate - lots of topics
+- Two suggestions:
+  - in PQCA TAC timeslot (alt. weeks) in off-week
+  - offline poll to accommodate most TSC members, and consider time rotation to handle differing timezones. This may require a higher frequency than monthly
+- Agreed poll will be conducted
+
 ## 8. Report from TAC representative (Thomas)
 
+- Project Lifecycle. 
+  - [Early draft](https://docs.google.com/document/d/1NV-0vNgXWdc81oqT0jv0C-9Funb8dySS06u90ghF-X4/edit)
+  - PQCA TAC suggests oqs tsc review and develop as most active project.
+  - objective is clarity for consumers as to what state the code is in - production, experimental, how good is security, maintenance - don't want to set wrong expectations. 
+  - Also covers adoption of new projects. Similar to apache incubation approach
+  - may apply at sub-project or component level
+  - should map existing oqs subprojects/components to proposal for validation
+  - clear separation may require feature flags, restructuring of liboqs, additional projects etc
+  - should delegate to OQS dev meeting to decide/discuss in github issues
+  - Issue will be opened for followup
```

Comment:
I think we start with a top level issue at the tsc level since it applies to the whole project (gh org).
I was going to raise an issue, but we have https://github.com/open-quantum-safe/tsc/issues/1 open already - are you ok we use that. Can update minutes with the link?